solver/llbsolver/file/backend.go

```diff
@@ -167,9 +176,14 @@ func rmPath(root, src string, allowNotFound bool) error {
 }
 
 func docopy(ctx context.Context, src, dest string, action pb.FileActionCopy, u *copy.User, idmap *idtools.IdentityMapping) error {
-	srcPath := cleanPath(action.Src)
-	destPath := cleanPath(action.Dest)
-
+	srcPath, err := cleanPath(action.Src)
+	if err != nil {
+		return errors.Wrap(err, "cleaning path")
```

Comment:
When wrapping errors, we should add more context than just the name of the function.
e.g.
```suggestion
		return errors.Wrap(err, "cleaning source path")
```